plugins/registry.js

```diff
@@ -461,4 +461,23 @@ export default class PluginRegistry {
 
         return {id, showRHSPlugin: showRHSPlugin(id)};
     }
+
+    // Register a component to replace code blocks with components provided by plugins.
+    // Accepts a React component. Returns a unique identifier.
```

Comment:
Can you document the `languages` parameter here?

Reply:
Done :+1: